simplified-viewer-audiobook/src/main/java/org/nypl/simplified/viewer/audiobook/AudioBookViewer.kt

```diff
@@ -38,17 +38,7 @@ class AudioBookViewer : ViewerProviderType {
     book: Book,
     format: BookFormat
   ) {
-    val formatAudio = format as BookFormat.BookFormatAudioBook
-    AudioBookPlayerActivity.startActivity(
-      from = activity,
-      parameters = AudioBookPlayerParameters(
-        formatAudio.manifest!!.manifestFile,
-        formatAudio.manifest!!.manifestURI,
-        book.id,
-        book.entry,
-        TODO()
-      )
-    )
+    TODO()
```

Comment:
Erk!

This is definitely wrong (as in the code was wrong before you replaced it with a single `TODO()` call). I need to go back and fill in the missing parameter, otherwise loading an audio book is going to result in a crash. :blush: